var/spack/repos/builtin/packages/r-pheatmap/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RPheatmap(RPackage):
+    """Implementation of heatmaps that offers more control over dimensions and
+    appearance."""
```

Comment:
Not strictly necessary, but the short description can be added as well.
```suggestion
    """Pretty Heatmaps
    
    Implementation of heatmaps that offers more control over dimensions and
    appearance."""
```